### PR TITLE
ENH - allowing a broadcast signal-based app termination

### PR DESCRIPTION
**Pull request by rksh**
from https://wigle.net/phpbb/viewtopic.php?f=3&p=9317#p9317

also removes some old intent handling code in MainActivity
tested using Tasker and adb.

### CODE REVIEW

File: wiglewifiwardriving/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -72,18 +72,6 @@
...
-                <category android:name="android.intent.category.DEFAULT" />
-            </intent-filter>
-            <intent-filter>
-                <action android:name="android.intent.action.ACTION_INSERT" />
-                <category android:name="android.intent.category.DEFAULT" />
-            </intent-filter>
-            <intent-filter>
-                <action android:name="android.intent.action.ACTION_DELETE" />
-                <category android:name="android.intent.category.DEFAULT" />
-            </intent-filter>
```
Comment (bobzilladev) [inline wiglewifiwardriving/src/main/AndroidManifest.xml]: I believe these are needed for the service-notifcation pause/scan buttons, no?

Comment (rksh) [inline wiglewifiwardriving/src/main/AndroidManifest.xml]: I thought these were the intent targets that got moved to ShareActivity per https://github.com/wiglenet/wigle-wifi-wardriving/blob/master/wiglewifiwardriving/src/main/AndroidManifest.xml#L175. WiGLEService maps them through ShareActivity at https://github.com/wiglenet/wigle-wifi-wardriving/blob/master/wiglewifiwardriving/src/main/java/net/wigle/wigleandroid/WigleService.java#L199  - seems to work in testing

Comment (rksh) [inline wiglewifiwardriving/src/main/AndroidManifest.xml]: (tested both from ListFragment menus and status badge pause/scan buttons)

Comment (bobzilladev) [inline wiglewifiwardriving/src/main/AndroidManifest.xml]: Ah, ok, looks right. as long as that's tested to still work. +1!

**Pull request merged**